app/stylesheet/application-webpack.scss

```diff
@@ -7,3 +7,19 @@
 @import '~@manageiq/react-ui-components/dist/textual_summary.css';
 @import '~@manageiq/react-ui-components/dist/tagging.css';
 @import '~@manageiq/ui-components/dist/css/ui-components.css';
+
+@import './application/main.scss';
```

Comment:
Let's name this folder something like legacy, or hacks ... because we will have to go through this and clean it up in the future anyway. I don't want to promote legacy stuff in the new world.

Reply:
:+1: Renamed :)